admin/bin/psychometricians-report-v2.js

```diff
@@ -23,41 +21,39 @@ const options = commandLineArgs(optionDefinitions)
 
 async function main (options) {
   try {
+    logger.info('force detected: re-processing all checks')
     if (options.forceReprocess) {
       // force the report to re-calculate the cached ps-report
-      winston.info('force detected: re-processing all checks')
+      logger.info('force detected: re-processing all checks')
```

Comment:
why log this twice?  surely just once after the detection of `options.forceReprocess`

Reply:
I suspect this was a bad merge or conflict resolution.  I've removed the line above that didn't make sense.